avocado_vt/plugins/vt_bootstrap.py

```diff
@@ -72,6 +72,27 @@ def configure(self, parser):
                             default=False, help=("All interactive "
                                                  "questions will be "
                                                  "answered with yes (y)"))
+        parser.add_argument("--vt-host-distro-name", action="store",
+                            metavar="HOST_DISTRO_NAME",
+                            help=("The name of the distro to be used when "
+                                  "generating the host configuration entry"))
+        parser.add_argument("--vt-host-distro-version", action="store",
+                            metavar="HOST_DISTRO_VERSION",
+                            help=("The version of the distro to be used when "
+                                  "generating the host configuration entry"))
+        parser.add_argument("--vt-host-distro-release", action="store",
+                            metavar="HOST_DISTRO_RELEASE",
+                            help=("The release of the distro to be used when "
+                                  "generating the host configuration entry."))
+        parser.add_argument("--vt-host-distro-arch", action="store",
+                            metavar="HOST_DISTRO_ARCH",
+                            help=("The architecture of the distro to be used when "
+                                  "generating the host configuration entry."))
+        parser.add_argument("--vt-host-distro-compat",
+                            action="store_true", default=False,
+                            help=("Whether to generate host distro "
+                                  "configuration suitable for use with current"
+                                  " test provider configuration"))
```

Comment:
This message is doomed to be outdated the moment this gets included as people (should) start using the mainstream way.

To be honest I don't think we should allow compat option as it only allows keeping the code gap and people would be confused as something will work properly with upstream, something with downstream and the code will get mixed and messy. I'd go with compat-mode only, or I'd simply initiate a big bang, implemented only non-compat versions and let downstream adapt (provide a sed script if you want to be nice).